[2025-03-15T17:46:18.671Z] error: Error: Invalid credentials here 2
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:32:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:11:45.452Z] error: Error: Invalid credentials
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:11:53.314Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:12:01.342Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:12:10.557Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:12:34.377Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:12:59.206Z] error: Error: Invalid credentials
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:13:07.498Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:13:32.573Z] error: Error: Invalid credentials
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:16:42.991Z] error: Error: Senha Incorreta
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:26:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:17:00.886Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:18:29.321Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:18:52.309Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:19:03.138Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:20:17.551Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:21:13.773Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:21:23.226Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:21:48.928Z] error: Error: Senha Incorreta
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:26:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:28:30.874Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-15T18:28:49.313Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:22:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:29:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-17T12:31:45.536Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:18:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:14:20)
[2025-03-17T14:28:53.447Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:17:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-18T17:04:45.814Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Can't reach database server at `localhost:5432`

Please make sure your database server is running at `localhost:5432`.
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-21T17:50:06.112Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Can't reach database server at `localhost:5432`

Please make sure your database server is running at `localhost:5432`.
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-21T17:50:23.730Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Error querying the database: FATAL: the database system is starting up
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-21T22:29:32.817Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Can't reach database server at `localhost:5432`

Please make sure your database server is running at `localhost:5432`.
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-22T14:39:08.476Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:21:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-23T15:38:08.835Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:21:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-23T17:34:49.825Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:21:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-24T12:21:13.523Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Can't reach database server at `localhost:5432`

Please make sure your database server is running at `localhost:5432`.
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-25T13:30:36.996Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Can't reach database server at `localhost:5432`

Please make sure your database server is running at `localhost:5432`.
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-26T13:11:54.864Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:17:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-26T13:11:55.025Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:21:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-27T12:48:37.340Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Can't reach database server at `localhost:5432`

Please make sure your database server is running at `localhost:5432`.
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-27T12:49:08.387Z] error: PrismaClientInitializationError: 
Invalid `prisma.user.findFirst()` invocation:


Error querying the database: FATAL: the database system is starting up
    at _n.handleRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:8049)
    at _n.handleAndLogRequestError (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:7057)
    at _n.request (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:121:6741)
    at async l (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\@prisma\client\runtime\library.js:130:9355)
    at async Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:16:27)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-27T19:20:50.375Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
[2025-03-27T19:21:06.074Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
[2025-03-27T19:22:22.542Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
[2025-03-27T19:35:21.349Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
[2025-03-27T19:36:11.119Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
[2025-03-27T19:45:03.721Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
[2025-03-27T20:20:11.233Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:21:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-30T17:47:24.210Z] error: Error: CPF ou Email não cadastrados.
    at Object.login (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/data-access/userRepo.ts:21:15)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:28:18)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-03-31T13:14:25.920Z] error: Error: Name, email, and senha are required.
    at Module.findSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/services/saldoService.ts:7:11)
    at userSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/controllers/saldoControllers.ts:11:39)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
[2025-03-31T13:16:59.497Z] error: Error: Name, email, and senha are required.
    at Module.findSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/services/saldoService.ts:7:11)
    at userSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/controllers/saldoControllers.ts:11:39)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
[2025-03-31T13:18:33.876Z] error: Error: Name, email, and senha are required.
    at Module.findSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/services/saldoService.ts:7:11)
    at userSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/controllers/saldoControllers.ts:11:39)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
[2025-04-01T13:53:35.973Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-01T13:54:58.799Z] error: Error: Name, email, and senha are required.
    at Module.findSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/services/saldoService.ts:7:11)
    at userSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/controllers/saldoControllers.ts:11:39)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
[2025-04-01T13:55:01.072Z] error: Error: Name, email, and senha are required.
    at Module.findSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/services/saldoService.ts:7:11)
    at userSaldos (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Saldos/controllers/saldoControllers.ts:11:39)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
[2025-04-01T14:33:17.592Z] error: Error: reCAPTCHA verification failed: DUPE
    at verifyRecaptcha (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/recaptcha-enterprise.ts:36:11)
    at async Module.loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/services/userService.ts:17:21)
    at async loginUser (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Users/controllers/userControllers.ts:13:20)
[2025-04-02T12:32:27.623Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-02T12:33:10.540Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-02T12:47:58.812Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-02T12:51:22.242Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-02T12:51:45.459Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-14T15:21:03.246Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-14T15:21:07.503Z] error: Error: function Error() { [native code] }
    at errorMiddleware (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/errors/errorMiddleware.ts:15:45)
    at Layer.handle_error (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15
    at Immediate.next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:265:14)
    at Immediate._onImmediate (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:513:21)
[2025-04-17T23:03:51.310Z] error: ReferenceError: dados is not defined
    at Module.fetchDates (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:7:40)
    at fetchDates (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:12:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
[2025-04-17T23:04:32.215Z] error: ReferenceError: res is not defined
    at Object.fetchDates (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:33:7)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.fetchDates (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:7:18)
    at async fetchDates (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:12:20)
[2025-04-18T19:47:39.983Z] error: ErrorID: w274gebb31k - next is not a function
[2025-04-18T19:48:13.943Z] error: ErrorID: 39uf8bxor22 - next is not a function
[2025-04-18T19:50:24.109Z] error: TypeError: schema.parse is not a function
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:47:12)
[2025-04-18T19:52:16.219Z] error: TypeError: schema.parse is not a function
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:47:12)
[2025-04-18T19:54:15.206Z] error: TypeError: schema.parse is not a function
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:47:12)
[2025-04-18T20:30:55.975Z] error: TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.util.objectKeys.Object.keys.keys [as objectKeys] (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:31:27)
    at ZodObject._getCached (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:2494:27)
    at ZodObject._parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:2509:49)
    at ZodObject._parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:2525:37)
    at ZodObject._parseSync (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:649:29)
    at ZodObject.safeParse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:679:29)
    at ZodObject.parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:660:29)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T20:40:00.462Z] error: TypeError: utilService.downloadFilesForCategories is not a function
    at downloadFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
[2025-04-18T20:47:41.217Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:88:13)
    at Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:29)
    at downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T20:50:20.567Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:88:13)
    at Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:29)
    at downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T20:54:51.111Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:95:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T20:56:30.158Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:95:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T20:56:40.385Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:95:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T20:56:51.347Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:95:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T20:58:45.454Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:95:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T20:59:58.118Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:95:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T21:29:51.640Z] error: TypeError: Cannot read properties of undefined (reading 'parse')
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:47:12)
[2025-04-18T21:32:06.400Z] error: TypeError: keyValidator._parse is not a function
    at ZodObject._parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:2525:37)
    at ZodObject._parseSync (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:649:29)
    at ZodObject.safeParse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:679:29)
    at ZodObject.parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:660:29)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T21:33:15.143Z] error: TypeError: keyValidator._parse is not a function
    at ZodObject._parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:2525:37)
    at ZodObject._parseSync (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:649:29)
    at ZodObject.safeParse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:679:29)
    at ZodObject.parse (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/node_modules/zod/lib/index.mjs:660:29)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T21:33:52.315Z] error: TypeError: The "paths[3]" argument must be of type string. Received undefined
    at Object.resolve (node:path:182:9)
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:105:26)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
[2025-04-18T21:34:49.677Z] error: TypeError: The "paths[3]" argument must be of type string. Received undefined
    at Object.resolve (node:path:182:9)
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:105:26)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
[2025-04-18T21:36:19.439Z] error: TypeError: The "paths[3]" argument must be of type string. Received undefined
    at Object.resolve (node:path:182:9)
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:105:26)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
[2025-04-18T21:39:34.305Z] error: TypeError: The "paths[3]" argument must be of type string. Received undefined
    at Object.resolve (node:path:182:9)
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:105:26)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
[2025-04-18T21:40:49.091Z] error: TypeError: files is not iterable
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:100:31)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T21:43:05.358Z] error: ReferenceError: fs is not defined
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:104:5)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T21:43:40.536Z] error: TypeError: fs.ensureDir is not a function
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:105:14)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T21:45:19.610Z] error: TypeError: The "path" argument must be of type string. Received undefined
    at Object.isAbsolute (node:path:446:5)
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:117:28)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
[2025-04-18T21:47:15.598Z] error: TypeError: Cannot destructure property 'fileName' of 'request' as it is undefined.
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:90:13)
    at Module.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:40:29)
    at unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T22:16:54.092Z] error: Error: ENOENT: no such file or directory, open 'C:\Users\lucas\Documents\PERSONAL\apidev-api\src\utils\downloader\downloaded\2025-03\Cnaes.zip'
[2025-04-18T22:16:54.095Z] error: Error: Failed to download files from Receita Federal
    at Object.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:83:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:29:18)
    at async downloadCategoriesFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:39:20)
[2025-04-18T22:18:10.927Z] error: Error: Invalid ZIP file
    at unzipFile (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/utils/unzipper/src/unzipper.ts:56:13)
    at async Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:123:7)
    at async unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:20)
[2025-04-18T22:18:10.932Z] error: Error: Failed to unzip file: Cnae
    at Object.unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:127:13)
    at async unzipFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:52:20)
[2025-04-18T22:18:23.410Z] error: ErrorID: 72rhukpkx3e - EISDIR: illegal operation on a directory, read
[2025-04-18T22:24:58.915Z] error: TypeError: Cannot read properties of undefined (reading 'parse')
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:47:12)
[2025-04-18T22:35:05.177Z] error: TypeError: Cannot read properties of undefined (reading 'parse')
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:16:14
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:47:12)
[2025-04-18T22:35:33.580Z] error: ReferenceError: path is not defined
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:50:24)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T22:36:58.256Z] error: ReferenceError: databaseImporter is not defined
    at Object.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:141:44)
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:51:27)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
[2025-04-18T22:38:13.249Z] error: Error: No schema defined for category: undefined
    at createTable (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/utils/importer/src/databaseImporter.ts:184:11)
    at processCategory (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/utils/importer/src/databaseImporter.ts:436:9)
    at Object.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:151:35)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:51:16)
    at async importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:20)
[2025-04-18T22:38:13.259Z] error: Error: Failed to import files for category "undefined"
    at Object.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:169:13)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:51:16)
    at async importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:20)
[2025-04-18T22:44:27.746Z] error: ReferenceError: path is not defined
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:57:24)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T22:45:00.374Z] error: ReferenceError: __dirname is not defined
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:59:5)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T22:45:30.298Z] error: ReferenceError: __dirname is not defined
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:59:5)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T22:46:39.662Z] error: ReferenceError: fileURLToPath is not defined
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:56:22)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\index.js:284:15
[2025-04-18T22:47:10.725Z] error: TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:141:26)
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:67:15)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
[2025-04-18T22:48:20.226Z] error: ReferenceError: fs is not defined
    at Object.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/data-access/utilsRepo.ts:158:19)
    at Module.importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/services/utilsServices.ts:67:15)
    at importFiles (file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/modules/Utilities/controllers/utilsControllers.ts:65:38)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at file:///C:/Users/lucas/Documents/PERSONAL/apidev-api/src/core/middlewares/validateRequest.ts:17:7
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (C:\Users\lucas\Documents\PERSONAL\apidev-api\node_modules\express\lib\router\layer.js:95:5)
